### ISSUE TITLE
Environment variables value escaped

### ISSUE THREAD

**Issue by oliveti** [cloudfoundry/nginx-buildpack#17]
What version of Cloud Foundry and CF CLI are you using? (i.e. What is the output of running `cf curl /v2/info && cf version`?

{
   "name": "",
   "build": "",
   "version": 0,
   "min_cli_version": "6.42.0",
   "min_recommended_cli_version": "latest",
   "api_version": "2.128.0",
   "osbapi_version": "2.14",
}
cf version 6.44.0+5de0f0d02.2019-05-01

What version of the buildpack you are using?

1.0.12

If you were attempting to accomplish a task, what was it you were attempting to do?

Replace environment variables in nginx.conf and return it.
Here is the config 👍 
```
     # configuration endpoint
    location /test {
      default_type application/json;

      set $override "{{env "OVERRIDE"}}";

      if ($override != '') {
          return 200 "{{env "OVERRIDE"}}";
      }

      try_files $uri /assets/test.json last;
    }
```

What did you expect to happen?

correct value to be returned : ` {   "abcd": 12345 }{   'ef': "ab" }`

What was the actual behavior?

html escaped value : `{   &#34;abcd&#34;: 12345 }{   &#39;ef&#39;: &#34;ab&#34; }`

Please confirm where necessary:
* [ ] I have included a log output
* [ ] My log includes an error message
* [ ] I have included steps for reproduction

Thank you

**Comment by tylerphelan** [comment]
@oliveti thanks for the issue, we'll look into this soon!

**Comment by dfreilich** [comment]
@oliveti Thanks for the request! This was fixed here: 9d14718af6fbb57cb1f1b1f5cf036facdf5e9ead
